From f60d0a69026fa42cf9e5a09f84da6680a35ac7ca Mon Sep 17 00:00:00 2001
From: Subhajit Paul <subhajit_paul@ti.com>
Date: Sun, 7 Jul 2019 16:52:31 +0530
Subject: [PATCH 2/6] ti_config_fragments: v8_baseport: Enable CONFIG_UIO

For Jailhouse inter VM communication, Jailhouse driver
adds a virtual PCI device in the kernel DT.
Enable CONFIG_OF_OVERLAY for this.

Also enable CONFIG_UIO to allow userspace IO to this
PCI device.

Signed-off-by: Subhajit Paul <subhajit_paul@ti.com>
Signed-off-by: Nikhil Devshatwar <nikhil.nd@ti.com>
---
 ti_config_fragments/v8_baseport.cfg | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ti_config_fragments/v8_baseport.cfg b/ti_config_fragments/v8_baseport.cfg
index 8ef9caa40008..afe692488fe1 100644
--- a/ti_config_fragments/v8_baseport.cfg
+++ b/ti_config_fragments/v8_baseport.cfg
@@ -159,3 +159,6 @@ CONFIG_TI_PAT=y
 
 # Watchdog support
 CONFIG_K3_RTI_WATCHDOG=m
+
+CONFIG_UIO=y
+CONFIG_OF_OVERLAY=y
-- 
2.17.1

